modules/tinymce/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The formatter can now apply a format to a `contenteditable="false"` element by wrapping it. Configurable using the `format_noneditable_selector` option. #TINY-8905
 
 ### Fixed
+- Fixed a regression where clicking in the editor would always show the link context toolbar when the option `link_context_toolbar` is set to true #TINY-8940
```

Comment:
This was never released to customers, so there was no regression to them and as such there should be no changelog entry.

```suggestion
```